### PR TITLE
Added 2023.9.5 CL

### PR DESCRIPTION
**Pull request by katrinDY**

### CODE REVIEW

File: src/modules/ROOT/pages/changelog/202309/05.json [hunk 1]
```diff
@@ -0,0 +1,110 @@
+{
+  "meta": {
+    "releasedOn": "2024.03.21"
+  },
+
+  "added": [
+    {
+      "what": [
+        "the option to access user information like username, handle, email, etc. in pages, charts, and record fields."
```
Comment (tjerman) [inline src/modules/ROOT/pages/changelog/202309/05.json]: Either provide the entire list of options or link to somewhere where all of the options are listed (like [a type reference](https://docs.cortezaproject.org/corteza-docs/2023.9/integrator-guide/expr/type-reference.html#objref-user))

File: src/modules/ROOT/pages/changelog/202309/05.json [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+      "what": [
+        "not being able to upload `.png` file to an application in {APP_NAME_ADMIN}."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/0456fe4f3e2e127dbccc76d131bbd3d706829133"
+      ]
+    },
+    {
+      "what": [
+        "not being able to make a copy of predefined CRM namespace."
```
Comment (tjerman) [inline src/modules/ROOT/pages/changelog/202309/05.json:61]: Don't think this was exclusive to the CRM namespace; think it was for all namespaces.
Best you double-check with Jože but I think that was the case

File: src/modules/ROOT/pages/changelog/202309/05.json [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/55a827a2cbd908f5d5bb573a85d145dfba7fa7f1",
+        "https://github.com/cortezaproject/corteza/pull/1729"
+      ]
+    },
+    {
+      "what": [
+        "not being able to add a record in a duplicated CRM namespace."
```
Comment (tjerman) [inline src/modules/ROOT/pages/changelog/202309/05.json]: Can you note down what exactly went wrong?

**Review by tjerman — CHANGES_REQUESTED** [review]
Almost there; good job!

File: src/modules/ROOT/pages/changelog/202309/05.json (hunk 2 above)
Comment (katrinDY) [reply src/modules/ROOT/pages/changelog/202309/05.json:61]: It's only for CRM. I double-checked

File: src/modules/ROOT/pages/changelog/202309/05.json [hunk 4]
```diff
@@ -6,7 +6,7 @@
   "added": [
     {
       "what": [
-        "the option to access user information like username, handle, email, etc. in pages, charts, and record fields."
+        "the option to access user information from link:https://docs.cortezaproject.org/corteza-docs/2023.9/integrator-guide/expr/type-reference.html#objref-user[User values] in pages, charts, and record fields."
```
Comment (tjerman) [inline src/modules/ROOT/pages/changelog/202309/05.json]: nono, use an `xref` (global search project to get an example); don't link to a static page as the link could break

**Pull request closed without merging**